aspnetcore/release-notes/aspnetcore-6.0.md

```diff
@@ -0,0 +1,704 @@
+      sec-fetch-site: [Redacted]
+      sec-fetch-mode: [Redacted]
+      sec-fetch-user: [Redacted]
+      sec-fetch-dest: [Redacted]
+info: Microsoft.AspNetCore.HttpLogging.HttpLoggingMiddleware[2]
+      Response:
+      StatusCode: 200
+      Content-Type: text/plain; charset=utf-8
+```
+
+The preceding output was enabled with the following *appsettings.development.json* file:
+
+[!code-json[](aspnetcore-6.0/samples/WebApp1/appsettings.development.json?highlight=6)]
+
+HTTP logging provides logs of:
+
+* HTTP Request information
+* Common properties
+* Headers
+* Body
+* HTTP Response information
+
+To configure the HTTP logging middleware, specify <xref:Microsoft.AspNetCore.HttpLogging.HttpLoggingOptions>:
+
+[!code-csharp[](aspnetcore-6.0/samples/WebApp1/Program.cs?name=snippet_httplg2&highlight=1,4-13)]
+
+### IConnectionSocketFeature
+
+The <xref:Microsoft.AspNetCore.Connections.Features.IConnectionSocketFeature> request feature provides access to the underlying accept socket associated with the current request. It can be accessed via the <xref:Microsoft.AspNetCore.Http.Features.FeatureCollection> on `HttpContext`.
+
+For example, the following app sets the <xref:System.Net.Sockets.Socket.LingerState> property on the accepted socket:
+
+[!code-csharp[](aspnetcore-6.0/samples/WebApp1/Program.cs?name=snippet_icsf)]
+
+### .NET Hot Reload
+
+Hot Reload minimizes the number of app restarts after code changes. For more information, see [.NET Hot Reload support for ASP.NET Core](https://docs.microsoft.com/aspnet/core/test/hot-reload)
+<!--TODO change above link to <xref:test/hot-reload> when https://github.com/dotnet/AspNetCore.Docs/pull/23659 merges-->
+
+### Generic type constraints in Razor
+
+When defining generic type parameters in Razor using the `@typeparam` directive, generic type constraints can now be specified using the standard C# syntax:
+
+
+### Smaller SignalR, Blazor Server, and MessagePack scripts
+
+ The SignalR, MessagePack, and Blazor Server scripts are now significantly smaller, enabling smaller downloads, less JavaScript parsing and compiling by the browser, and faster start-up. The size reductions:
+
+* `signalr.min.js`: 70%
+* `blazor.server.js`: 45%
+
+The smaller scripts are a result of a community contribution from [Ben Adams](https://twitter.com/ben_a_adams). For more information on the details of the size reduction, see [Ben's GitHub pull request](https://github.com/dotnet/aspnetcore/pull/30320).
+
+### Enable Redis profiling sessions
+
+A community contribution from [Gabriel Lucaci](https://twitter.com/_glucaci) enables Redis profiling session with [Microsoft.Extensions.Caching.StackExchangeRedis](https://www.nuget.org/packages/Microsoft.Extensions.Caching.StackExchangeRedis):
+
+[!code-csharp[](aspnetcore-6.0/samples/WebApp1/Program.cs?name=snippet_red)]
+
+For more information, see [StackExchange.Redis Profiling](https://stackexchange.github.io/StackExchange.Redis/Profiling_v2.html).
+
+### Shadow copying in IIS
```

Comment:
@shirhatti  @jkotalik  please review Shadow copy section.

> Please leave feedback on IIS Shadow copying in [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/23733).

 Can you provide a GitHub issue for feedback? The doc's repo is probably the wrong place.